### ISSUE TITLE
[SUGGESTION] Rename HAIntegration::configure() to either HAIntegration::begin() or HAIntegration::setup() 

### ISSUE THREAD

**Issue by JacobChrist** [daniloc/PicoW_HomeAssistant_Starter#16]
Rename HAIntegration::configure() to either HAIntegration::begin() or HAIntegration::setup() for more orthogonal naming with the Wiring/Arduino naming conventions.

I feel like begin() better matches other Arduino classes in how its used, but because of the loop() it really feels like it should be called setup().

Frankly, configure() doesn't really bother me either, but I think begin() or setup() really hits home as to what this function is doing and where it probably should be called from.

**Comment by daniloc** [comment]
Yeah, thanks for thinking about this, I agree. `begin()` is probably the most culturally idiomatic. You've got a few thoughts brewing, want to open a PR to reflect this change?
